src/tests/Blef.GameLogic.Tests/PokerHandsHierarchy/PokerHandHierarchyTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Blef.GameLogic.PokerHands;
+using Blef.GameLogic.Tests.TestData;
+using NUnit.Framework;
+
+namespace Blef.GameLogic.Tests
+{
+    public class PokerHandHierarchyTest
+    {
+        [TestCaseSource(typeof(TestPokerHandsGenerator), nameof(TestPokerHandsGenerator.GeneratePokerHandsMatrix))]
+        public static void should_be_able_to_tell_which_one_pokerhand_is_stronger_than_another(int i, int j,
```

Comment:
It covers all possible poker hands combinations. GeneratePokerHandsMatrix create a Cartesian product of all possible poker hands ordered by hierarchy.  Bug #25 is also covered.
```plaintext
 should_be_able_to_tell_which_one_pokerhand_is_stronger_than_another(21,22,TwoPairs of King and Queen,TwoPairs of Ace and Nine) [0:00.000] Success
```
When I run this test first time it failed. Now it is passing (after fixing #25)